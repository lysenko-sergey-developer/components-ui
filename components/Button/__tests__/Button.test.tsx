import * as React from 'react'
import { shallow } from 'enzyme'

import { Button } from '..'

test('CheckboxWithLabel changes the text after click', () => {
  const button = shallow(<Button />)

  // Snapshot demo
  expect(button).toMatchSnapshot(<div>111</div>)
})
