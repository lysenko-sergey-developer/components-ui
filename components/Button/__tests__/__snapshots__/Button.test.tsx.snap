// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button test cases Button onClick is valid callback 1`] = `
Object {
  "text": "secondState",
}
`;

exports[`Button test cases Button was success rendered! 1`] = `ShallowWrapper {}`;
